docs/introduction/what-is-tendermint.md

```diff
@@ -299,8 +299,8 @@ introduces a few **locking** rules which modulate which paths can be
 followed in the flow diagram. Once a validator precommits a block, it is
 locked on that block. Then,
 
-1.  it must prevote for the block it is locked on
-2.  it can only unlock, and precommit for a new block, if there is a
+1. it must prevote for the block it is locked on
+2. it can only unlock, and precommit for a new block, if there is a
     polka for that block in a later round
```

Comment:
I don't know if we want to eliminate the term of polka in use and just use polc as that is the terminology used in the code

Reply:
yeah I'm not sure if we want to do this now or another time but it would be nice to streamline the terminology a little bit